apps/comps/hooks/useUnlockKeys.ts

```diff
@@ -18,45 +19,39 @@ export const useUnlockKeys = (agentHandle: string, agentId?: string) => {
     enabled: !!agentId,
   });
 
-  // Query for sandbox agent - fetch when agent exists
-  const sandboxAgentQuery = useQuery({
-    queryKey: ["sandbox-agent", agentHandle],
-    queryFn: async () => {
-      return await sandboxClient.getAgentApiKey(agentHandle);
-    },
-    enabled: !!agentHandle,
-  });
-
-  // Get sandbox agent id
```

Comment:
`sandboxAgentQuery` is redundant. the rest of this logic is simply moved after the `sandboxKeyQuery` because we'll get or create an agent in that step